db/migrate/20240208111830_remove_repo_column_from_application.rb

```diff
@@ -0,0 +1,9 @@
+class RemoveRepoColumnFromApplication < ActiveRecord::Migration[7.1]
+  def up
+    remove_column :applications, :repo
+  end
+
+  def down
+    add_column :applications, :repo, :string
+  end
```

Comment:
I know this is just a draft, and you may have caught this later on, but just in case: your `down` method doesn't add the removed index back.

You could also simplify this migration by using `change` instead of `up` and `down` (both `remove_index` and `remove_column` are reversible).

(Untested:)

```suggestion
  def change
    remove_index :applications, column: :repo
    remove_column :applications, :repo, :string
  end
```

Reply:
Thanks @robinjam !